packages/chatbot-eval-mongodb-public/src/eval.config.ts

```diff
@@ -20,9 +20,13 @@ const {
   MONGODB_DATABASE_NAME,
   MONGODB_CONNECTION_URI,
   CONVERSATIONS_SERVER_BASE_URL,
+  OPENAI_CHAT_COMPLETION_DEPLOYMENT,
+  OPENAI_ENDPOINT,
+  OPENAI_API_KEY,
 } = assertEnvVars(envVars);
 
 export default async () => {
+  const { OpenAIClient, AzureKeyCredential } = await import("@azure/openai");
```

Comment:
Why use a dynamic import instead of static at the top of the file?

Reply:
commonjs / esm funkiness. 

we're using common.js and the package is esm only. this is the work around.